src/pages/Login/Login.test.js

```diff
@@ -70,7 +70,7 @@ describe("<Login /> view", () => {
     )
   })
 
-  it.skip("calls SIGNUP_MUTATION without error", async () => {
+  it.only("calls SIGNUP_MUTATION without error", async () => {
```

Comment:
remove only

Reply:
should I change it back to skip? I haven't worked with test cases before.

Reply:
it's all good, didn't realize dave had already merged. i'll take care of it.

Reply:
I fixed another issue in the same file and changed it back in that one.

Reply:
Thanks for reviewing @harpalassi 😊

Reply:
> should just be like `it("calls ...`
> `only` will make it so the other tests won't run anymore.
Oops, I added skip back 🤦‍♂️ in that PR. Will fix that before merging.

Reply:
nice, to clarify - you don't need to change it back to skip. just "it" by itself is what is needed here.

Reply:
Done!